model/activist.go

```diff
@@ -1131,14 +1145,17 @@ func UpdateActivistData(db *sqlx.DB, activist ActivistExtra, userEmail string) (
 	// TODO: make this work for other chapters too, maybe based on Chapters table mailing list info?
 	// or maybe just passing chapter Name or ID to the signup service?
 	if origActivist.ChapterID == 47 {
-		activistInfoChanged := activist.Name != origActivist.Name ||
+		geoInfoChanged := activist.City != origActivist.City ||
+			activist.State != origActivist.State ||
+			activist.StreetAddress != origActivist.StreetAddress
+		mailingListInfoChanged := activist.Name != origActivist.Name ||
 			activist.Email != origActivist.Email ||
 			activist.Phone != origActivist.Phone ||
 			activist.Location != origActivist.Location ||
```

Comment:
would be nice to move this down a couple lines underneath the state != state check to keep things in the same order that they are used later on, to more easily verify correctness at a glance.